src/main/java/uk/ac/ebi/atlas/trader/GxaExperimentRepository.java

```diff
@@ -54,13 +58,16 @@ public Experiment getExperiment(String experimentAccession) {
         switch (experimentDto.getExperimentType()) {
             case PROTEOMICS_BASELINE:
             case RNASEQ_MRNA_BASELINE:
-                return baselineExperimentFactory.create(experimentDto, experimentDesign, idfParserOutpout);
+                return baselineExperimentFactory.create(experimentDto, experimentDesign, idfParserOutpout,
```

Comment:
I just noticed that we have a typo in `idfParserOutpout`. Cold you change it to `idfParserOutput`, please?

Reply:
Fixed.